src/promptflow-evals/promptflow/evals/evaluators/_groundedness/_groundedness.py

```diff
@@ -49,7 +49,10 @@ def __call__(self, *, answer: str, context: str, **kwargs):
         :rtype: dict
         """
         # Validate input parameters
-        if not (answer and answer.strip()) or not (context and context.strip()):
+        answer = str(answer or "")
```

Comment:
Is this same as what dag flows were doing ?

Reply:
Yes, with this change, the behavior will be same as before.